articles/overview/understanding-quantum-computing.md

```diff
@@ -0,0 +1,60 @@
+Quantum physics, also called quantum mechanics or quantum theory, is a branch of physics that deals with particles at the atomic and subatomic level. At the quantum level, however, many of the laws of physics you take for granted don’t apply. Superposition, quantum measurement, and entanglement are three phenomena that are central to quantum computing.  
+
+### Superposition vs. binary computing
+Anyone with a hectic work or school schedule knows that you can’t be two places at once – you have to be either *here* or *there*. However, if you’re a quantum particle then you can be *here* AND *there* AND *somewhere else* due to a phenomenon known as **superposition** (also known as **coherence**).
+
+Think of flipping a coin in the air and catching it. While it’s in the air turning over and over, it’s neither heads nor tails, but can be seen as a linear combination of all of its possible states at once – heads, tails, 75% heads, 12.5% tails, and so on.
+
+A quantum particle such as an electron has its own “heads or tails” property called **spin**, which can be either up or down (or positive or negative for a photon), or to make it more relatable to classical binary computing, let’s just say 1 or 0. When a quantum particle is in a superposition state, it’s a linear combination of all of its possible states between 1 and 0, but you don’t know which one it will be until you actually look at it, which brings up our next phenomenon, **quantum measurement**.
+
+### Quantum measurement
+
+Let’s say you want to track the height of a five-year old child every year by measuring them and placing a mark on the wall. You would expect their height to increase every year, and the mark on the wall to gradually get higher.
+
+But what if you measured that five-year old and they just stopped growing? That’s sort of what happens when you measure a quantum particle. Just the act of observing or measuring a quantum particle **collapses** the superposition state (also known as **decoherence**) and the particle takes on a classical binary state of either 1 or 0.
+
+This is helpful to us, because in computing you can do lots of things with 1’s and 0’s. However, once a quantum particle has been measured and collapsed, it stays in that state forever and will always be a 1 or 0, and cannot be used for more calculations. As you’ll see later, though, in quantum computing there are operations that can “reset” a particle back to a superposition state so it can be useful again.
+
+### Entanglement
+
+Possibly the most interesting phenomenon of quantum physics is the ability of two or more quantum particles to become **entangled** with each other. When particles become entangled, they form a single system such that the quantum state of any one particle cannot be described independently of the quantum state of the other particles. This means that whatever operation or process you apply to one particle immediately takes effect on the other particles as well.
```

Comment:
> This means that whatever operation or process you apply to one particle immediately takes effect on the other particles as well.

This could be easily misunderstood as claiming that faster-than-light causality is possible, but that's not the case. I'd suggest being a bit more clear about what's meant by "effect" in this case.